api/src/opentrons/protocol_api/module_contexts.py

```diff
@@ -105,6 +105,7 @@ def load_labware_object(self, labware: Labware) -> Labware:
                 deck_slot=deck_slot,
                 on_module=True,
                 offset_id=provided_offset.offset_id,
+                labware_display_name=labware.label,
```

Comment:
Will this raise if the requested PAPI version is less than 2.13? Should we reach into the private `labware._implementation` instead to make sure everything still works even if the user hasn't opted into v2.13?

```suggestion
                labware_display_name=labware._implementation.get_label(),
```

Reply:
Even the `get_label` method is added in this PR. There was no way to fetch just the label from the labware..